website/src/pages/index.js

```diff
@@ -0,0 +1,112 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from "react";
+import classnames from "classnames";
+import Layout from "@theme/Layout";
+import Link from "@docusaurus/Link";
+import useDocusaurusContext from "@docusaurus/useDocusaurusContext";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import styles from "./styles.module.css";
+
+const features = [
+  {
+    title: <>Cross Platform</>,
+    imageUrl: "img/xplat.png",
```

Comment:
This will break when we deploy on the live site because the baseUrl needs to be included. Wrap in `useBaseUrl()` here

Reply:
`imageUrl` of these two features wrapped with `useBaseUrl()` at the time of mapping:

```js
{features && features.length && (
  <section className={styles.features}>
    <div className="container">
      <div className="row">
        {features.map(({ title, imageUrl, description }, idx) => (
          <div
            key={idx}
            className={classnames("col col--6", styles.featureBlock)}
          >
            {imageUrl && (
              <div className="text--center">
                <img
                  className={styles.featureImage}
                  src={useBaseUrl(imageUrl)}
                  alt={title}
                />
              </div>
            )}
            <h3>{title}</h3>
            <p>{description}</p>
          </div>
        ))}
      </div>
    </div>
  </section>
)}
```